### ISSUE TITLE
--lw & font color issue

### ISSUE THREAD

**Issue by xradiation** [n3oxmind/gns3theme#8]
Hi,
There is an issue with the `--lw` link width flag, since `setyp.py` is complining about `#` in front of the link width value. This can be solved by omitting the `#` at line 381

```diff
        --lw)
            linkFlag=true
-           colorscheme[10]="#${2}"
+           colorscheme[10]="${2}"
            shift 2
            ;;
```

Another thing is that am not able to change the font color in the gns3 preview window, say like the text on top of network switches, routers ... I've tested all flag combinations with no luck so far; Is it possible to share the `n30x-dark3` theme config since am clearly missing somthing here.

Best Regards.

**Comment by n3oxmind** [comment]
@xradiation , thanks for reporting this issue. I'll make sure to fix it.

> Is it possible to share the n30x-dark3 theme config since am clearly missing somthing here.

its called `n30x-dark` now. you can install it like this.
` ./gns3theme.sh -u $USER --install --scheme n30x-darkw`

>Another thing is that am not able to change the font color in the gns3 preview window, say like the text on top of network switches, routers ... I've tested all flag combinations with no luck so far


 I have removed this feature. You can still change it from `Preferences->General->Default label style`.
![image](https://user-images.githubusercontent.com/10103340/103386459-c767c480-4ab3-11eb-8247-7ef1ec5d66af.png)

Please note that, new label style will be applied for the new appliance (routers, switches, ..., etc) only. Already existing appliance will have to be changed manually ( right-click ).

I've updated the script to apply  the link and grid colors automatically to fit the colorscheme. 

Please download and test on your end.
https://github.com/n3oxmind/gns3theme.git

thanks,

**Comment by xradiation** [comment]
Applying the link & grid colors without explicitly setting the `--lc & --gc` flags makes much more sense, since the user is usually is looking for a consistent look & feel.

Right after posting this issue I've been able to figure out how to setup pretty much everything I needed, I've also added a custom color scheme based on `n30x-darker` :

```yaml
dwm-dark:         #0d0d0d #141414 #ffffff #00b6ff #181818 #000000 #BA4551 #161616 #b3b3b3 #6e6e6e 2   #303030   dark
```
And BTW. I really appreciate you work @n3oxmind , this project is really helpful since by default gns3 isn't that eyes friendly at all.
